### DISCUSSION TITLE
Changed a setting and don't know where it is - Column names in result panel have spaces

### DISCUSSION THREAD

**Discussion by Austin1** [dbeaver/dbeaver#15606] [Q&A]
I did something yesterday and now when the result panel appears it is adding Spaces where underscores were in the column names. The database does not actually include spaces. I changed something but cannot find for the life of me what box I ticked.

**Comment by LonwoLonwo** [comment]
Hello @Austin1 

Can you provide some examples? Pictures can be beneficial.
